applications/processor/groovy-processor/pom.xml

```diff
@@ -14,10 +14,6 @@
         <relativePath>../../stream-applications-core/pom.xml</relativePath>
     </parent>
 
-    <properties>
-        <apache-ivy.version>2.5.1</apache-ivy.version>
```

Comment:
Sorry, Chris, how does it work if we don’t specify version any more?
Is this Spring Boot managed dep?
So, it comes as latest now and with all those CVEs fixed?

Reply:
Good point @artembilan - I was not clear in my description. We (stream-applications) already have dep. mgmt. for this ivy in [stream-applications-build](https://github.com/spring-cloud/stream-applications/blob/e9afea157b1a8e3e83a2de1fa250d70111594b05/stream-applications-build/pom.xml#L108-L112) but for some reason these versions were still being hardcoded in these modules. 
This moves those versions out of the way and lets the stream-applications-build/pom.xml be the final arbiter of the versions.